src/condor_daemon_core.V6/daemon_core.cpp

```diff
@@ -8052,6 +8052,7 @@ int DaemonCore::Create_Process(
 				}
 				else if (errno == ENOENT)
 				{
+					TemporaryPrivSentry tps(PRIV_USER);
```

Comment:
This needs to be guarded by priv==PRIV_USER(_FINAL). This will EXCEPT() if we're trying to launch a condor-owned process and the user priv account hasn't been initialized.